src/py/flwr/cli/new/templates/app/pyproject.flwrtune.toml.tpl

```diff
@@ -0,0 +1,44 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "$package_name"
+version = "1.0.0"
+description = ""
+authors = [
+    { name = "The Flower Authors", email = "hello@flower.ai" },
+]
+license = { text = "Apache License (2.0)" }
+dependencies = [
```

Comment:
I think `sentencepiece` is needed (but please check if that's the case on a clean new env -- on my side a `ValueError` was thrown because of it)